### DISCUSSION TITLE
Name! Give it a name!

### DISCUSSION THREAD

**Discussion by Sebfrom17** [Blackymas/NSPanel_HA_Blueprint#413] [General]
Hi there,

I am from yesterday a happy beginner user of Your project. One of the most fantastic ideas around HA for a long time! Thank You for Your efforts, and keep going! I have a small suggestion - could You give Your project a friendly name? Something unique, maybe even commercial. It would be easier for us users to find related materials on the web, as the project and the ideas around it will grow, which I certainly believe it will!

**Comment by Blackymas** [comment]
Hi and thanks

what do you mean? New Name hear in github?

**Reply by Sebfrom17** [reply]
Hi, I mean something like "WLED" project or anything similar, kind of a hashtag. Yes, You can start by adding the name on a Github. When I try, for example, to find videos related to Your project on YT, I get everything around NSP, NSP pro, different blueprints, and so on. Your project is so brilliant (it should become an official HA addon or integration) - that it deserves a unique name.
